src/vs/platform/extensionManagement/node/extensionManagementService.ts

```diff
@@ -160,7 +160,7 @@ export class ExtensionManagementService extends Disposable implements IExtension
 
 			const manifest = await getManifest(zipPath);
 			const identifier = { id: getGalleryExtensionId(manifest.publisher, manifest.name) };
-			// let operation: InstallOperation = InstallOperation.Install; {{ SQL CARBON EDIT }}
+			// let operation: InstallOperation = InstallOperation.Install; {{SQL CARBON EDIT}}
 			// {{SQL CARBON EDIT}}
```

Comment:
remove?

Reply:
I presume the one underneath is for the Azure Data Studio extension error